fvm/evm/evm.go

```diff
@@ -12,39 +12,48 @@ import (
 	"github.com/onflow/flow-go/model/flow"
 )
 
-func RootAccountAddress(chainID flow.ChainID) (flow.Address, error) {
+func ContractAccountAddress(chainID flow.ChainID) (flow.Address, error) {
 	sc := systemcontracts.SystemContractsForChain(chainID)
 	return sc.EVM.Address, nil
```

Comment:
Maybe also rename `EVM` to `EVMContract`, just like the EVM storage account is called `EVMStorage`